app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,31 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@drawable/wallpaper_main_screen"
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/title_text"
```

Comment:
please use the naming convention <WHAT>_<WHERE>_<DESCRIPTION> use more descriptive names

Reply:
done